services/API-service/src/scripts/json/layer-metadata.json

```diff
@@ -77,7 +77,7 @@
         "drought": "This layer displays the probability that seasonal precipitation will fall within the lowest third (lower tercile) of the 30-year climatological distribution."
       },
       "LSO": {
-        "drought": "This layer displays the probability that seasonal precipitation will fall within the lowest third (lower tercile) of the 30-year climatological distribution."
+        "drought": "This layer displays the probability that seasonal precipitation will fall within the lowest third range of the 30-year climatological distribution.<br/><br/>The high/low rainfall forecast for the current period does not directly determine the number of people exposed to or affected by the forecasted drought but can be determined by comparing the forecasted rainfall with what is expected based on the last 30 years of climatology for each area.<br/><br/>Source: <a target=“_blank” href='https://charts.ecmwf.int/products/seasonal_system5_standard_rain?area=GLOB&base_time=202503010000&stats=tsum&valid_time=202504010000'>European Centre for Medium-Range Weather Forecasts (ECMWF)</a>"
```

Comment:
what are the weird-looking double quotes around _blank? In practice single quotes work as well (and is used at other places). Anyway, if this works also fine.

Reply:
You're right, they should be single quotes.